### PR TITLE
[FLINK-9299] ProcessWindowFunction documentation Java examples have errors

### PR DESCRIPTION
**Pull request by yanghua**
## What is the purpose of the change

*This pull request fixed some document demo code error.*

## Brief change log

  - *This pull request fixed some document demo code error.*

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / **no**)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / **no**)
  - The serializers: (yes / **no** / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / **no** / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (yes / **no** / don't know)
  - The S3 file system connector: (yes / **no** / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / **no**)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / **not documented**)

### CODE REVIEW

**Comment by yanghua** [comment]
cc @zentol @tillrohrmann

**Comment by yanghua** [comment]
@StephanEwen does this has any problem need to change? it seems @medcv try to fixed this issue before asking me.

**Comment by medcv** [comment]
@yanghua Thanks for the fix. I will close my PR as you addressed all the issues in the ticket here.

**Comment by sihuazhou** [comment]
hi @medcv  I think maybe it better to assign the issue to yourself before working on that, this could help to avoid depulicate works(cause i notice that you make the PRs without assigning to yourself), you can request the contribution permisson from the dev mail, i think PMCs will accpet your request once they saw your request, it always very quickly. With the permisson, you can assign the issue as you wish then.

Best~

File: docs/dev/stream/operators/windows.md [hunk 1]
```diff
@@ -883,7 +883,7 @@ private static class AverageAggregate
 
   @Override
   public Double getResult(Tuple2<Long, Long> accumulator) {
-    return accumulator.f0 / accumulator.f1;
+    return (double) accumulator.f0 / accumulator.f1;
```
Comment (fhueske) [inline docs/dev/stream/operators/windows.md]: Should probably be `((double) accumulator.f0) / accumulator.f1;`

**Comment by fhueske** [comment]
Thanks for the fix @yanghua. I left a minor comment. 
Otherwise +1 to merge.

**Comment by yanghua** [comment]
cc @fhueske changed and fixed a remained error before

**Comment by yanghua** [comment]
cc @tzulitai @fhueske

File: docs/dev/stream/operators/windows.md [hunk 2]
```diff
@@ -797,7 +797,7 @@ DataStream<SensorReading> input = ...;
 
 input
   .keyBy(<key selector>)
-  .timeWindow(<window assigner>)
+  .timeWindow(<time size>)
```
Comment (StephanEwen) [inline docs/dev/stream/operators/windows.md]: How about using "duration" instead of "time size". I think "time size" is not a commonly used term...

Comment (yanghua) [reply docs/dev/stream/operators/windows.md]: accept, hold on...

**Comment by yanghua** [comment]
@StephanEwen fixed based on your suggestion

**Comment by StephanEwen** [comment]
Looks good, thanks, merging this...

**Pull request closed without merging**